src/EA.Weee.Web/Areas/AatfReturn/Mappings/ToViewModel/ReturnToObligatedViewModelMap.cs

```diff
@@ -60,7 +60,7 @@ public ObligatedViewModel Map(ReturnToObligatedViewModelMapTransfer source)
             }
             else if (source.OperatorName != null)
             {
-                transfer.WeeeDataValues = source.ReturnData.ObligatedWeeeSentOnData.Where(w => w.Aatf.Id == source.AatfId).ToList();
+                transfer.WeeeDataValues = source.ReturnData.ObligatedWeeeSentOnData.Where(w => w.WeeeSentOnId == source.WeeeSentOnId).ToList();
```

Comment:
Same for this one, if the change was the result of page create a unit test to ensure it is always set